tests/test_pipenv.py

```diff
@@ -796,3 +796,23 @@ def test_local_zipfiles(self):
             dep = p.lockfile['default'][key]
 
             assert 'file' in dep or 'path' in dep
+
```

Comment:
This test fails on windows. We need it to work before merging.

Reply:
This might actually be a bug on windows in pipenv due to how windows opens and places files locks on things.

Reply:
could this be the problem?
https://bugs.python.org/msg27445

it says the file is probably locked by the process
we could probably unlock the file or something?